render/detailed_node.go

```diff
@@ -95,49 +110,35 @@ func MakeDetailedNode(r report.Report, n RenderableNode) DetailedNode {
 		}
 	}
 
-	tables := tables{}
+	// Connections table
 	{
-		rows := []Row{}
+		cRows := []Row{}
```

Comment:
Ah I see.  You should move the sort up to where the connections rows are populated.  Also, rename the rows type to rowSort or something; thats a much smaller rename.